hs_discover/public/index.html

```diff
@@ -71,9 +71,8 @@ <h2 class="page-title">Discover
                 <div class="panel panel-default">
                     <div class="panel-heading" role="tab" id="headingLegend">
                         <h4 class="panel-title">
-                            <span class="glyphicon glyphicon-info-sign"></span>&nbsp;
-                            <a role="button" data-toggle="collapse" data-parent="#accordion" href="#legend-collapse"
-                               aria-expanded="true" aria-controls="legend-collapse">Legend</a>
+                            <a role="button" style="text-decoration: none" data-toggle="collapse" data-parent="#accordion" href="#legend-collapse"
+                               aria-expanded="true" aria-controls="legend-collapse"><span class="glyphicon glyphicon-info-sign"></span> Legend</a>
                         </h4>
```

Comment:
include the glyphicon in the clickable anchor

Reply:
It'd be good to put ```text-decoration: none``` into css rather than using style here in the template.

Reply:
The CSS includes have a lot of sprawl and some conflict. I'd like to leave this for now, as it's very clear what is being applied and after the includes are moved and the scoped CSS is applied, where relevant the dust will settle on these types of inline styling.